installer/docker/README.md

```diff
@@ -1,81 +1,109 @@
+6. Inside here, make sure that all Ubuntu options are toggled. Then press apply & restart.
+   
+![image](https://github.com/jonescompneurolab/hnn/assets/34087669/d01a0c14-37b2-456a-b7d8-3ed3bdd5283c)
 
-```bash
-docker build --tag jonescompneurolab/hnn:win64 -f installer/windows/Dockerfile installer/docker
-```
+8. To check if Docker is correctly installed, you can open up the Ubuntu terminal (application) and type `docker --version`, If done correctly, a version and build number of the software installed should be fed back.
 
-## Starting HNN container
 
-The container is designed to be run from the `hnn_docker.sh` script, but in general, the following scheme will work for Linux/mac:
+### Part 3 Install Vcxsrv
+1. Please install Vcxsrv https://sourceforge.net/projects/vcxsrv/
+2. Run the installer, choosing `C:\Program Files\VcXsrv` as the destination folder.
+3. A new icon named `XLaunch` should now appear on your computer. Click on the icon to open up the application and continue with the installation steps.
+4. Choose "Multiple windows" for `Select display settings`. Choose '0' for the `Display number`. Click 'Next'.
 
-```bash
-docker run -d -v "$HOME/hnn_out":"$HOME/hnn_out" --env XAUTHORITY=/tmp/.Xauthority --env SYSTEM_USER_DIR="$HOME" --name hnn_container jonescompneurolab/hnn
-```
+![Screenshot 2023-11-29 163402](https://github.com/jonescompneurolab/hnn/assets/34087669/89bcad1a-56fd-4026-a718-ea3c81b8554d)
 
-This will start the container in the background. In order to start the HNN GUI, the start_hnn.sh script must be run inside the container:
+5. Select "Start no client" and click 'Next'.
 
-## Running HNN with Docker
+![Screenshot 2023-11-29 164111](https://github.com/jonescompneurolab/hnn/assets/34087669/57ba748c-6a05-424d-aa7b-8afcc7f9b917)
 
-We recommend using the `hnn_docker.sh` script which includes checks for directory permissions, creating the necessary files, and adding the appropriate options for each host OS. The commands below give an outline of the process.
+7. Please make sure "Disable access control" is checked and click 'Next'. This will take you to the final page where you can press 'Finish' to complete the installation.
 
-Using SSH can be controlled by the environment variable USE_SSH
+![Screenshot 2023-11-29 163550](https://github.com/jonescompneurolab/hnn/assets/34087669/d4c7341b-1b00-4721-b49d-79b5dd081047)
 
-### Without SSH (only recommended for Linux)
 
-```bash
-export USE_SSH=0
-docker exec --env SYSTEM_USER_DIR="$HOME" --env DISPLAY=":0" -u "$(id -u)" hnn_container /home/hnn_user/start_hnn.sh
+### Part 4 Configure and Run HNN Application
+1. Open up a new Ubuntu terminal application, referenced in Step 3 of Part 1. (Note: ensure that the Ubuntu terminal is being used and not the Windows Powershell)
+2. We will now download the source code that starts up the GUI. Paste this command to create a copy of the source code on your computer.
+```
+git clone https://github.com/jonescompneurolab/hnn.git
```

Comment:
@dylansdaniels I see, so this line here needs to be changed to something like:
```suggestion
git clone --single-branch --branch maint/pre-hnn-core https://github.com/jonescompneurolab/hnn.git
```

Reply:
Yes exactly!

Reply:
Thank you for the review, change commited